test/cbmc/proofs/skipScalars/Makefile

```diff
@@ -0,0 +1,25 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+EXPENSIVE = 1
```

Comment:
This will work, but I've noticed in other projects we set it to `true`.

https://github.com/aws/aws-encryption-sdk-c/pull/710/files

Reply:
Implemented as suggested.